src/pages/TableWithTabs.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+    <f-table
+        :tabs-data="headerTabs"
+        header-help-text="Payment requests"
+    >
+        <template v-slot:header-right>
+            <q-toggle
+                v-model="selection"
+                left-label
+                color="blue"
+                label="Auto Payment"
+                val="auto_payment" />
+
+            <q-toggle
+                v-model="selection"
+                left-label
+                color="blue"
+                label="Payment Request"
+                val="auto_payment" />
+        </template>
+
+        <template v-slot:control-right>
+            <div class="q-gutter-xs q-ma-none">
+                <q-btn
+                    color="primary"
+                    label="On Left" />
+                <q-btn
+                    color="secondary"
+                    label="On Right" />
```

Comment:
Use translations on the texts/labels?

Reply:
that was for demo purposes that's why I didn't use it. Now using translation.